<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Введите свои данные для регистрации в системе&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Фамилия</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Имя</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отчество</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Логин</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Пароль</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Повтор пароля</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>Open Sans</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="font">
        <font>
         <family>Open Sans</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="font">
        <font>
         <family>Open Sans</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="font">
        <font>
         <family>Open Sans</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="font">
        <font>
         <family>Open Sans</family>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="font">
        <font>
         <family>Open Sans</family>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(48, 48, 222);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
